### ISSUE TITLE
No trailing whitespace should ignore multiline template strings

### ISSUE THREAD

**Issue by feross** [eslint/eslint#5786]
**What version of ESLint are you using?**

2.2.0

**What parser (default, Babel-ESLint, etc.) are you using?**

default

**Please show your full configuration:**

```
{
  "no-trailing-spaces": 2
}
```

**What did you do? Please include the actual source code causing the issue.**

```javascript
let str = `
${a}
      <-- significant trailing whitespace
${b}
`
```

**What did you expect to happen?**

Whitespace could be significant inside multiline template strings, so the rule should ignore it.

**What actually happened? Please include the actual, raw output from ESLint.**

```
t.js:3:1: Trailing spaces not allowed. (no-trailing-spaces)
```

**Comment by platinumazure** [comment]
I'm unable to repro due to the ESLint online demo not supporting ES6 yet, but if this is confirmed, I think this can safely be called a bug.

**Comment by mysticatea** [comment]
Thank you for this issue.
I confirmed.

@platinumazure The online demo is supporting ES6. There is a configuration panel at the bottom of the page.

**Comment by kaicataldo** [comment]
I can take a look at this one
